domino/domino.py

```diff
@@ -1203,6 +1217,17 @@ def _validate_blob_key(key):
                 )
             )
 
+    @staticmethod
+    def _validate_blob_path(path):
```

Comment:
would love some test and a docstring explaining what this does?

Reply:
added! :)